BackofficeBundle/Manager/NodeManager.php

```diff
@@ -92,6 +97,20 @@ public function createNewLanguageNode(NodeInterface $node, $language)
     }
 
     /**
+     * @param NodeInterface $node
+     *
+     * @return StatusInterface
+     */
+    protected function getInitialStatus(NodeInterface $node)
```

Comment:
I use this beacause I need the initial status of the node, not the status which is flagged initial.
But I change it. @nicolasThal